src/mudblazor/MudBlazor.Template/Components/Layout/NavMenu.razor

```diff
@@ -1,21 +1,57 @@
-﻿<MudNavMenu>
+﻿@*#if (IndividualLocalAuth)
+@implements IDisposable
+
+@inject NavigationManager NavigationManager
+
+##endif*@
+
+<MudNavMenu>
     <MudNavLink Href="" Match="NavLinkMatch.All" Icon="@Icons.Material.Filled.Home">Home</MudNavLink>
-@*#if (UseServer || UseWebAssembly)
+    @*#if (UseServer || UseWebAssembly)
     <MudNavLink Href="counter" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Filled.Add">Counter</MudNavLink>
-##endif*@
+    ##endif*@
     <MudNavLink Href="weather" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Filled.List">Weather</MudNavLink>
+    @*#if (IndividualLocalAuth)
+    <MudNavLink Href="auth" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Filled.List">Auth Required</MudNavLink>
+    <AuthorizeView>
+        <Authorized>
+            <MudNavLink Href="Account/Manage" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Filled.Add">@context.User.Identity?.Name</MudNavLink>
+            <form action="Account/Logout" method="post">
+                <AntiforgeryToken />
+                <input type="hidden" name="ReturnUrl" value="@currentUrl" />
+                <MudButton Type="ButtonType.Submit">Logout</MudButton>
```

Comment:
How do I make a `MudNavLink` submit a form?   This is the "Logout" link when authenticated.